templates/js-template-default/frameworks/runtime-src/proj.android-studio/gradle.properties

```diff
@@ -17,10 +17,26 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-PROP_TARGET_SDK_VERSION=22
+# Android SDK version that will be used as the compile project
+PROP_COMPILE_SDK_VERSION=27
+
+# Android SDK version that will be used as the earliest version of android this application can run on
+PROP_MIN_SDK_VERSION=16
```

Comment:
what min sdk version and app platform version you used to build external android libs? set it 16, compile success, 14 ,compile failed.

`PROP_MIN_SDK_VERSION=16` means min sdk version and app platform version both are 16, you can search External Native Build log to verify this.